api/base/serializers.py

```diff
@@ -977,6 +977,11 @@ def resolve_url(self, obj, request):
             raise SkipField
         if self.must_be_file is True and obj.path.endswith('/'):
             raise SkipField
+
+        if 'view_only' not in self.kwargs:
+            view_only = request.query_params.get('view_only', False)
+            self.kwargs['view_only'] = view_only
```

Comment:
This looks like it is going to add `view_only=False` to all wb links that don't have vols, is that intentional?

Reply:
Oh no, it's not intentional. I will update the PR to fix that.